docs/source/index.rst

```diff
@@ -7,16 +7,51 @@ ChainerCV is a **deep learning based computer vision library** built on top of `
 
 
 Install Guide
--------------
+=============
+
+Pip
+~~~
 
-Please install `Chainer <https://github.com/pfnet/chainer/>`_ first.
 You can install ChainerCV using `pip`.
 
 .. code-block:: shell
 
+    # If Cython has not been installed yet, install it by commands like
```

Comment:
I wanted it to imply that `pip install Cython` is not the only way to install Cython.
People install them using apt as an alternative to pip for instance.